app/src/main/java/org/oppia/app/home/UserAppHistoryViewModel.kt

```diff
@@ -31,4 +31,4 @@ class UserAppHistoryViewModel @Inject constructor(
     }
     return appHistoryResult.getOrDefault(UserAppHistory.getDefaultInstance())
   }
-}
+}
```

Comment:
EOF missing

Reply:
Also, I think there are no changes in this file, can you make it up-to-date with develop?

Reply:
resolved